### PR TITLE
Add about/ecosystem

### PR DESCRIPTION
**Pull request by lognaturel**
This is the start of an ecosystem page.

While it could certainly live under "software", I think putting it under "about" makes it about the people and organizations as much as about the software. I think it also makes it seem a bit more important (and I think it is).

### CODE REVIEW

File: _pages/about/ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+permalink: /about/ecosystem/
+sidebar:
+  nav: "about"
+---
+
+There exists a broad community of organizations that contribute to or deploy core ODK tools and develop some of their own compatible tools that can be mixed and matched. This ecosystem means that organizations can use the tools that best suit their needs for any given project and do not need to worry about vendor lock-in.
+
+The main components that make tools in the ecosystem compatible are the [XForms form specification](https://opendatakit.github.io/xforms-spec/) and the [OpenRosa standards](https://docs.opendatakit.org/openrosa/). Ecosystem members collaborate on these shared components.
+
+## Ecosystem tools
```
Comment (yanokwa) [inline _pages/about/ecosystem.md]: I would prefer as a first pass to list folks who are open source. To that end, I'd drop SurveyCTO and add SMAP.

File: _pages/about/ecosystem.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* [ELMO](http://getelmo.org/), hosted server solution originally designed for election monitoring
+* [Enketo](https://enketo.org), web application for filling forms
+* [GeoODK](http://geoodk.com/), derived tools with additional geo capabilities
+* [Kobo](http://www.kobotoolbox.org/), hosted server with graphical form builder
+* [Medic mobile](https://medicmobile.org), hosted server for frontline medical care
```
Comment (yanokwa) [inline _pages/about/ecosystem.md]: I think they like Mobile capitalized.

File: _pages/about/ecosystem.md (hunk 1 above)
Comment (lognaturel) [reply _pages/about/ecosystem.md]: Done.

File: _pages/about/ecosystem.md (hunk 2 above)
Comment (lognaturel) [reply _pages/about/ecosystem.md]: Done.

**Pull request merged**